### PR TITLE
localizedCallerName added to displayIncomingCall

### PR DESCRIPTION
**Pull request by aarkalyk**
Added localizedCallerName for displaying a callerName, so that handle can be used for storing caller id/number. Please, review the PR. 
Thanks!

### CODE REVIEW

File: ios/RNCallKit/RNCallKit.m [hunk 1]
```diff
@@ -84,8 +84,9 @@ - (void)dealloc
 // Display the incoming call to the user
 RCT_EXPORT_METHOD(displayIncomingCall:(NSString *)uuidString
                                handle:(NSString *)handle
-                               handleType:(NSString *)handleType
-                             hasVideo:(BOOL)hasVideo)
+                           handleType:(NSString *)handleType
+                             hasVideo:(BOOL)hasVideo
+                  localizedCallerName:(NSString * _Nullable)localizedCallerName))
```
Comment (ianlin) [inline ios/RNCallKit/RNCallKit.m]: Have got an extra `)` at the end, please fix it.

**Comment by aarkalyk** [comment]
My bad😬 Just fixed it!

**Comment by ianlin** [comment]
Thanks! react-native-callkit 1.3.2 released.

**Comment by sujameslin** [comment]
@aarkalyk Can you please add `localizedCallerName` support to `startCall` method too?

**Comment by aarkalyk** [comment]
@sujameslin Sure 👌

**Comment by sujameslin** [comment]
@aarkalyk Did you manage to add `localizedCallerName` to `startCall` method?

**Comment by aarkalyk** [comment]
@sujameslin turns out there's no `localizedCallerName` in `CXStartCallAction` class. There's `contactIdentifier` instead, which seems like to be used for same purposes. Here's what apple's docs say about it:

> The identifier is displayed in the native call UI, and is typically the name of the call recipient.

**Pull request merged**